### PR TITLE
Add user test 1 resources!

### PR DESCRIPTION
**Pull request by isabela-pf**
Outlining user testing resources and results from test 1: navigation!

This PR adds
- The `user-tests` directory
- A directory readme with a list of links to all other resources in the directory
- The `0-support` directory for resources that are not test specific, like recruiting email templates our or potential test notebooks
- The `1-navigation` directory with the notebook we tested in, the test script, and a summary of the results

This might look like a lot to review, but there are a lot of unordered lists, I promise. 😆 Please let me know if I can help with the review process as I think this very much needs feedback. I'd also love to find ways to include any info you find missing in a way that keeps our participants' privacy.

### CODE REVIEW

File: user-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Resources not tied to any single test round.
+
+- [Outreach and recruitment email template](0-support/outreach-email-template.md)
+
+## Test 1: Navigation
+
+August 2022. How do users navigate and access different levels of structure within HTML outputs of Jupyter notebooks?
+
+- [Test script](1-navigation/test-script.md)
+- [Test notebook ()
```
Comment (isabela-pf) [inline user-tests/README.md:14]: Add note describing this notebook! Makes it more approachable and clear!

**Comment by isabela-pf** [comment]
While I would like to add more with more feedback from @/Jenneh when she is available, I'm marking this as ready for review. Now that a majority of the content is there, I'd be comfortable merging this and adding more later on as we see fit.

Hooray! 🌻

File: user-tests/README.md (hunk 1 above)
Comment (isabela-pf) [reply user-tests/README.md]: [Done in this commit](https://github.com/Iota-School/notebooks-for-all/pull/26/commits/4a88aa4077394951d9e9f54107473ebb8a3334a9). Thank you for the feedback @smythp! And let me know if this needs more context!

**Review by Jenneh — APPROVED** [review]
This is FANTASTIC!

My only small edit would be to remove "but it was worth noting." on line 105 of the results doc

**Pull request merged**